### ISSUE TITLE
Document make wasi-libc

### ISSUE THREAD

**Issue by fgsch** [tinygo-org/tinygo#1678]
This is not mentioned anywhere I can see. It would be a good idea to document it.

**Comment by aykevl** [comment]
Please take a look at https://github.com/tinygo-org/tinygo/pull/1679 which should make the error message a lot clearer and more helpful. For example, I got this error message after removing the libc:

```
$ go install; and tinygo run -target=wasi ./testdata/cgo
error: could not find wasi-libc, perhaps you need to run `make wasi-libc`?
```

That's a lot more helpful than a linker error.

**Comment by fgsch** [comment]
@aykevl I will take a look. Thank you.

Can we still document this step in the building instructions?

**Comment by aykevl** [comment]
Done: https://github.com/tinygo-org/tinygo-site/pull/150

**Comment by fgsch** [comment]
Re #1679, works as advertised. Thank you!

**Comment by deadprogram** [comment]
This has been released with v0.17.0 so now closing. Thanks!
